includes/template-library/classes/class-import-images.php

```diff
@@ -120,6 +134,21 @@ public function import( $attachment, $parent_post_id = null ) {
 			return false;
 		}
 
+		$filetype = wp_check_filetype( $filename );
+
+		if ( 'svg' === $filetype['ext'] ) {
+			if ( ! self::$svg_handler ) {
+				self::$svg_handler = new Svg_Handler();
+			}
+
+			// In case that unfiltered-files upload is not enabled, SVG images should not be imported.
+			if ( ! self::$svg_handler->is_enabled() ) {
+				return false;
+			}
+
+			$file_content = self::$svg_handler->sanitizer( $file_content );
+		};
+
```

Comment:
Change according to the new `Uploads_Manager` changes

Reply:
Done.